interfaces/builtin/udisks2.go

```diff
@@ -352,6 +353,15 @@ KERNEL=="sr*", ENV{ID_VENDOR}=="SanDisk", ENV{ID_MODEL}=="Cruzer", ENV{ID_FS_LAB
 ENV{ID_PART_TABLE_TYPE}=="dos", ENV{ID_PART_ENTRY_TYPE}=="0x0", ENV{ID_PART_ENTRY_NUMBER}=="1", ENV{ID_FS_TYPE}=="iso9660|udf", ENV{UDISKS_IGNORE}="0"
 `
 
+const udisks2PermanentSlotUDevTag = `
+# This file contains udev rules for UDisks2 service.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="sr*|md*|fd*|sd*|mmcblk[0-9]|mspblk[0-9]", TAG+="%[1]s"
+SUBSYSTEMS=="usb", TAG+="%[1]s"
```

Comment:
I know we're missing vd*. I think perhaps change this to:
```
SUBSYSTEM=="block", TAG+="%[1]s"
KERNEL=="fd[0-9]*, TAG+="%[1]s"
# This tags all USB devices, so we'll use AppArmor to mediate specific access (eg, /dev/sd* and /dev/mmcblk*)
SUBSYSTEM=="usb", TAG+="%[1]s"
```

Can you update the apparmor policy to have this:
```
/dev/vd* r,
```